appcli/app_test.go

```diff
@@ -166,14 +167,14 @@ func TestGetConfigMetadata_defaults(t *testing.T) {
 		)
 	}
 
-	if metadata.Quiet {
+	if metadata.Verbosity == ui.VerbosityLevelQuiet {
```

Comment:
Rather than just changing this to pass, make sure to preserve the purpose of the original test case.

This was making sure that the metadata did not reflect a quiet verbosity level for normal output, so this check and the corresponding error message should now be something like:

```go
if metadata.Verbosity != ui.VerbosityLevelNormal {
	t.Errorf(
  		"GetConfigMetadata(%s): expected normal verbosity: %v, actual: %v",
  		args, ui.VerbosityLevelNormal, metadata.Verbosity,
 	)
}
```